.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: opendistroforelasticsearch/security-maven:v1
+
+    steps:
+
+    - name: Checkout security-parent
+      uses: actions/checkout@v1
+      with:
+        repository: opendistro-for-elasticsearch/security-parent
+        ref: refs/heads/master
```

Comment:
Is this always checking out master? Suppose someone raises a PR against security opendistro-1.1 branch and security-parent master has the code for opendistro-1.3, how will the action work in this case?

Reply:
The user must modify this workflow file in the PR for testing. Point to refs/head/branchname or tag or commit

Reply:
I see. Then we should also update the Readme or another appropriate place to reflect this.